catalyst-gateway/Earthfile

```diff
@@ -80,6 +81,14 @@ build:
     SAVE ARTIFACT target/doc doc
     SAVE ARTIFACT target/release/cat-gateway cat-gateway
 
+test-openapi-diff:
```

Comment:
Looks good, but not sure whether it is worth moving this compare thing to cat-ci, since we are moving out of earthly + only use here

Reply:
I dont think that there is some benefit in this case to move this into the cat-ci.